pkg/apis/pipeline/v1/pipelinerun_defaults_test.go

```diff
@@ -168,6 +210,9 @@ func TestPipelineRunDefaulting(t *testing.T) {
 				TaskRunTemplate: v1.PipelineTaskRunTemplate{
 					ServiceAccountName: "tekton",
 				},
+				Timeouts: &v1.TimeoutFields{
+					Pipeline: &metav1.Duration{Duration: 5 * time.Minute},
```

Comment:
where does 5 come from here? is this the default timeout?

Reply:
This is set by the withContext Data setup, to make it more readable, I modified this into a int const.